lib/utils/getInterestOptions.ts

```diff
@@ -23,5 +23,8 @@ export type InterestType = (typeof interests)[number];
 
 export const getInterestOptions = () => {
   // returning mutable type array instead of `Read Only` type.
-  return [...interests];
+
+  const isOctober = new Date().getMonth() === 9;
+
+  return isOctober ? [...interests] : interests.filter((interest) => interest !== "hacktoberfest");
```

Comment:
Does this logic line up with how we treat Hacktoberfest @jpmcb? I know we have an environment variable for the year for Hacktoberfest, but I could've sworn we activated Hacktoberfest sometime after day one? Maybe I'm hallucinating. 😅

Reply:
I don't think we need this logic. Let's just remove `hacktoberfest` from the default options

Reply:
FYI - in the API, we would set and redeploy the hacktoberfest "year" environment variable to denote which year of data we're pulling in. So it should mostly be on the API side but a moot point if we're removing it.